ports/stm/boards/stm32f411ce_blackpill_haxpress/mpconfigboard.mk

```diff
@@ -0,0 +1,18 @@
+USB_VID = 0x239A
+USB_PID = 0x806A
+USB_PRODUCT = "stm32f411ce blackpill"
+USB_MANUFACTURER = "Unknown"
+
+SPI_FLASH_FILESYSTEM = 1
+EXTERNAL_FLASH_DEVICE_COUNT = 1
+EXTERNAL_FLASH_DEVICES = W25Q64FV #See supervisor/shared/external_flash/devices.h for options
```

Comment:
Don't put the comment on the same line: it can cause weird issues. Thanks.
If you know the chip type on the commercial version, change the `_COUNT` to 2, and add the other device, like
`= "W25Q64FV,blahblah"`